Index.html

```diff
@@ -0,0 +1,144 @@
+        <!-- Card One -->
+        <div class="card">
+          <div class="icon">
+            <i class="fa-solid fa-person-chalkboard"></i>
+          </div>
+          <div class="title">
+            <h4>Teachers</h4>
+          </div>
+          <div class="description">
+            <p>We have trained proffessionals who are mostly examiners.</p>
+          </div>
+        </div>
+          <!-- Card two -->
+          <div class="card">
+            <div class="icon">
+              <i class="fa-solid fa-people-group"></i>
+            </div>
+          
+          <div class="title">
+            <h4>Campers</h4>
+          </div>
+          <div class="description">
+            <p>We have trained proffessionals who are mostly examiners.</p>
+          </div>
+        </div>
+          <!-- Card three -->
+          <div class="card">
+            <div class="icon">
+              <i class="fa-solid fa-book-open"></i>
+              </div>
+              <div class="title">
+                <h4>Courses</h4>
+              </div>
+              <div class="description">
+                <p>We have trained proffessionals who are mostly examiners.</p>
+              </div>
+            </div>
+              <!-- Card four -->
+              <div class="card">
+                <div class="icon">
+                  <i class="fa-solid fa-person-hiking"></i>
+                </div>
+                <div class="title">
+                  <h4>Extras</h4>
+                </div>
+                <div class="description">
+                  <p>We have trained proffessionals who are mostly examiners.</p>
+                </div>
+              </div>
+                <!-- Card five -->
+                <div class="card">
+                  <div class="icon">
+                    <i class="fa-solid fa-globe"></i>
+                   </div>
+                  <div class="title">
+                    <h4>Countries</h4>
+                  </div>
+                  <div class="description">
+                    <p>We have trained proffessionals who are mostly examiners.</p>
+                  </div>
+                </div>
+      </div>
+                <button type="button" id="reg">Register for Free</button>
+    </section>
+    <!-- Teachers info section -->
+    <section id="teachers-section">
+      <h2 class="facilitators">Facilitators</h2>
+      <div id="facilitators-persons">
+        
+      </div>
+    </section>
+  </main>
+  <!-- Sponsors section  -->
+  <footer id="sponsor-sections">
+    <h3>Patners</h3>
+    <hr id="line">
+    <div id="sponsors-container">
+      <img src="./Assets/SLALogo.png" alt="SLA logo">
+      <img src="./Assets/linkedin.jpg" alt="linkedin logo">
+      <img src="./Assets/youtube.jpg" alt="youtube logo">
+      <img src="./Assets/facebook.jpg" alt="facebook logo">
+      <img src="./Assets/google.jpg" alt="google logo">
+      <p>Hope Alive Foundation</p>
+    </div>
+  </footer>
```

Comment:
- Please, kindly restructure your footer on both the `index.htm.` and `about.html`. As the both footer should be the same.
- Also, as you'e use the footer tag, according to the design, this should be a section with it contents [images and header(as you've used) ] with the correct background, so that you can add a footer separately with a white background and it's content, esuring the style matches the design as much as you can.
- As of now, your footer doesn't match the design, please kindly update the HTML structure for both the inde.htm and about.html.
- Lastly, kindly remove the added paragraph as this is not added in the original design.

*see the design*
![f_sec](https://user-images.githubusercontent.com/59397159/176675458-520f4169-9f58-4600-a13b-a6fe2730497d.png)

:point_right: *After updating it, you should have you design match the image above, featuring the  partners section and footer added and should be the same in both the `index.html` and `about.html` page*